tests/bits/parameter_handler_backslash_06.expect=diff.output

```diff
@@ -0,0 +1,3 @@
+
+DEAL::a, b, c
+DEAL::a, b, c
```

Comment:
The file has ``expect=diff`` so I assume the parser crashes and the test passes before getting to the diff stage? That is why I was hoping for a comment in the .cc.